src/go/rpk/pkg/cli/cmd/common/common.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package common
+
+import "fmt"
+
+const FeedbackMsg = `We'd love to hear about your experience with redpanda:
+https://vectorized.io/feedback`
+
+func DeprecationMessage(altCmd string) string {
+	return fmt.Sprintf("use '%s' instead.", altCmd)
```

Comment:
NIT: I would add some date or revision to the deprecation message when the command will be removed.

Reply:
> for example @0x5d I think this should come w/ a table.

I updated the PR description.

Reply:
Thanks for the suggestion! Per our discussion on slack, they will be removed when all the dependencies are updated.